modules/n1ql/pages/n1ql-language-reference/with-recursive.adoc

```diff
@@ -226,6 +501,17 @@ WITH RECURSIVE similar_items AS (
 SELECT * FROM similar_items;
 ----
 
+.Results
```

Comment:
This is fine as-is, but for this one example I think it would be fine just to say "this query will run until the Query service runs out of memory".